index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <link rel="stylesheet" href="styles.css">
+        <title>first mobile app</title>
+    </head>
+
+    <body>
+        <section class="main-page">
+            <header class="main-header">
+                <nav>
+                    <div class="header-text">John Doe</div>
+                </nav>
+                <nav class="union">
+                    <div class="vector"></div>
+                    <div class="vector"></div>
+                    <div class="vector"></div>
+                </nav>
+            </header>
+            
+        <div class="page1">
+            <h1 class="main-title">Hey There.<br>I'm Alex</h1>
+            <h2 class="job-title">I'm a Software Developer</h2>
+            <h5 class="regular-text">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                don’t hestiate to contact me.</h5>
+                <div class="social-media">
+                    <nav class="icon">
+                        <a href="https://github.com/mohisa302">
+                            <img src="icon1.png" alt="github icon"></a>
+                    </nav>
+                    <nav class="icon">
+                        <a href="https://www.linkedin.com/in/rachelisaac13">
+                            <img src="icon2.png" alt="linkedin icon"></a>
+                    </nav>
+                                <nav class="icon">
+                                    <a href="#">
+                    <img src="icon3.png" alt="github icon"></a>
+                </nav>
+                <nav class="icon">
+                    <a href="https://twitter.com/Kholochelam">
+                        <img src="icon4.png" alt="twitter icon"></a>
+                    </nav>
+                    <nav class="icon">
+                        <a href="#">
+                            <img src="icon5.png" alt="Medium icon"></a>
+                        </nav>
```

Comment:
- [ ] This nav should be a ul then it should have the a tags in the li each. Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using href which you have done but the ul and li is left.

Reply:
done